### ISSUE TITLE
[Request] Make xhr response available on Uploader @upload event

### ISSUE THREAD

**Issue by micheleangioni** [quasarframework/quasar#384]
Hi again,

at the moment the Uploader @upload event just passes the uploaded file name to the event itself.
It would be much more useful to have **the full response** available

```html
<q-uploader
    :url="/my-url"
    @upload="postUpload"
></q-uploader>
```

```js
methods: {
  postUpload (event) {
    console.log(event) // event is just the file name! The full response would be more useful
  }
}
```

Would that be feasable? :)

**Comment by rstoenescu** [comment]
Yep. And it's very easy too. Will do it right after I finish up with the new forms work item.

**Comment by micheleangioni** [comment]
Great! Looking forward to it!

**Comment by micheleangioni** [comment]
@rstoenescu could you please add this to 0.14 milestone? 
Thanks :)

**Comment by rstoenescu** [comment]
Added. You're right.

**Comment by rstoenescu** [comment]
Available in edge version and future v0.14.

**Comment by micheleangioni** [comment]
Great! :+1:
